<launch>

	<node name="odom_est" pkg="ironbot_rmt_ctrl" type="odom_est.py"/>
	<node name="console" pkg="ironbot_rmt_ctrl" type="console.py" />

	<node name="motion_ctrl" pkg="ironbot_rmt_ctrl" type="scan_capture.py" />

</launch>
